solutions/fully-configurable/outputs.tf

```
@@ -1,6 +1,33 @@
 ##############################################################################
 # Outputs
 ##############################################################################
+##############################################################################
+#  Monitoring and Workload Protection agent- Next Steps URLs outputs
+##############################################################################
+
+output "next_steps_text" {
+  value       = "Your Monitoring agent is ready to send metrics to the instance."
+  description = "Next steps text"
+}
+
+output "next_step_primary_label" {
+  value       = "Go to Cloud Monitoring dashboard"
+  description = "Primary label"
+}
+
+output "next_step_primary_url" {
+  value       = "https://cloud.ibm.com/observability/embedded-view/monitoring/${element(split(":", var.instance_crn), 7)}"
+  description = "Primary URL"
+}
+
+output "next_step_secondary_label" {
+  value       = "Learn more about IBM Cloud Monitoring"
```

```suggestion
  value       = "Learn more about working with metrics"
```

This is the title of the docs page.